src/main/webapp/view/transcript/transcript-area.js

```diff
@@ -21,9 +21,12 @@ export default class TranscriptArea {
   static #TRANSCRIPT_CONTAINER = 'transcript-lines-container';
   static #PARAM_ID = 'id';
 
+  static #transcriptLineMap = new Map();
```

Comment:
Why is this static? 

Also, remember not to add redundant names like "Map". This is a map of transcriptLine to comment count maybe rename it transcriptLineToCommentCount